frontend/src/views/camp/Invitation.vue

```diff
@@ -1,15 +1,15 @@
 <template>
-  <auth-container v-if="ready">
-    <div v-if="invitationFound">
-      <h1 v-if="userDisplayName" class="display-1">
-        {{ $tc('components.invitation.userWelcome') }} {{ userDisplayName }}
```

Comment:
This was here on purpose to welcome the user by his name (if logged in).
Before this was the possibility used to check if the user was logged in or not.

Also from the background that the recipient of an invitation may use another account to accept the invitation.

Now you see your name in the navigation, so we can remove userDisplayName if we want to use this way.